pyproject.toml

```diff
@@ -1,12 +1,11 @@
-[tool.poetry]
+[project]
 name = "kpops"
 version = "9.1.0"
 description = "KPOps is a tool to deploy Kafka pipelines to Kubernetes"
-authors = ["bakdata <opensource@bakdata.com>"]
-license = "MIT"
+authors = [
+  { name = "bakdata", email = "opensource@bakdata.com" },
+]
 readme = "README.md"
-repository = "https://github.com/bakdata/kpops"
-documentation = "https://bakdata.github.io/kpops/latest"
```

Comment:
These are missing now, also `license`

Reply:
yes, these two are Poetry-specific and don't exist in the offical pyproject.toml spec. I went with the recommendation to set MIT license as classifier instead since it's standardized.